packages/cmrjs/package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@cumulus/common": "1.11.2",
     "got": "^8.3.0",
+    "js2xmlparser": "^4.0.0",
```

Comment:
js2xmlparser handles mappings to XML strings in a handy fashion.   This is to solve the bug being discussed by @ifestus  re: CUMULUS-668, as js2xmlparser will generate a xml string from a hashmap, preserving order, and CMR's ECHO10 ingest requires the OnlineResources/OnlineAccessUrls to be in a particular order, however admittedly we need to consider refactoring to avoid usage of multiple xml libraries.      

Avoiding doing that here, in lieu of review comments and/or it being addressed elsewhere.   It *is* required for the CMR integration tests to pass.

Reply:
Discussed this this morning with @ifestus  @flamingbear, will write a separate ticket to resolve/refactor prior to closing the ticket, absent any objections here.